tests/regression/test_csm.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+
+from brownie import reverts, web3
+
+from utils.config import (
+    contracts,
+    ContractsLazyLoader,
+)
+from utils.dsm import UnvetArgs, to_bytes, set_single_guardian
+from utils.evm_script import encode_error
+from utils.test.csm_helpers import csm_add_node_operator, get_ea_member, csm_upload_keys
+from utils.test.deposits_helpers import fill_deposit_buffer
+from utils.test.helpers import ETH
+from utils.test.oracle_report_helpers import oracle_report
+from utils.test.staking_router_helpers import set_staking_module_status, StakingModuleStatus
+
+contracts: ContractsLazyLoader = contracts
+
+CSM_MODULE_ID = 3
+
+
+@pytest.fixture(scope="module")
+def csm():
+    return contracts.csm
+
+
+@pytest.fixture(scope="module")
+def accounting():
+    return contracts.cs_accounting
+
+
+@pytest.fixture(scope="module")
+def fee_distributor():
+    return contracts.cs_fee_distributor
+
+
+@pytest.fixture()
+def node_operator(csm, accounting) -> int:
+    address, proof = get_ea_member()
+    return csm_add_node_operator(csm, accounting, address, proof)
+
+
+@pytest.fixture()
+def pause_modules():
+    # pause deposit to all modules except csm
+    # to be sure that all deposits go to csm
+    modules = contracts.staking_router.getStakingModules()
```

Comment:
why not use the `getStakingModule` method instead of looping through all modules?

```
 function getStakingModule(uint256 _stakingModuleId)
        public
        view
        returns (StakingModule memory)
    {
        return _getStakingModuleById(_stakingModuleId);
    }
```

Reply:
copy-paste from sdvt tests
seems fine for me to iterate through all the modules here, we need to pause all of them except csm